src/cmake/axom-config.cmake.in

```diff
@@ -0,0 +1,131 @@
+  set(AXOM_USE_CUDA           "@AXOM_USE_CUDA@")
+  set(AXOM_USE_MPI            "@AXOM_USE_MPI@")
+  set(AXOM_USE_OPENMP         "@AXOM_USE_OPENMP@")
+
+  # Axom components
+  set(AXOM_ENABLE_LUMBERJACK  "@AXOM_ENABLE_LUMBERJACK@")
+  set(AXOM_ENABLE_MINT        "@AXOM_ENABLE_MINT@")
+  set(AXOM_ENABLE_PRIMAL      "@AXOM_ENABLE_PRIMAL@")
+  set(AXOM_ENABLE_QUEST       "@AXOM_ENABLE_QUEST@")
+  set(AXOM_ENABLE_SIDRE       "@AXOM_ENABLE_SIDRE@")
+  set(AXOM_ENABLE_SLAM        "@AXOM_ENABLE_SLAM@")
+  set(AXOM_ENABLE_SLIC        "@AXOM_ENABLE_SLIC@")
+  set(AXOM_ENABLE_SPIN        "@AXOM_ENABLE_SPIN@")
+
+  # Axom built-in TPLs
+  set(AXOM_USE_CLI11          "@AXOM_USE_CLI11@")
+  set(AXOM_USE_FMT            "@AXOM_USE_FMT@")
+  set(AXOM_USE_SPARSEHASH     "@AXOM_USE_SPARSEHASH@")
+
+  # Axom external TPLs
+  set(AXOM_USE_CONDUIT        "@AXOM_USE_CONDUIT@")
+  set(AXOM_USE_HDF5           "@AXOM_USE_HDF5@")
+  set(AXOM_USE_MFEM           "@AXOM_USE_MFEM@")
+  set(AXOM_USE_RAJA           "@AXOM_USE_RAJA@")
+  set(AXOM_USE_SCR            "@AXOM_USE_SCR@")
+  set(AXOM_USE_UMPIRE         "@AXOM_USE_UMPIRE@")
+
+  #----------------------------------------------------------------------------
+  # Bring in required  dependencies for this axom configuration
+  #----------------------------------------------------------------------------
+  include(CMakeFindDependencyMacro)
+
+  # conduit
+  if(AXOM_USE_CONDUIT)
+    set(AXOM_CONDUIT_DIR  "@CONDUIT_DIR@")
+    if(NOT CONDUIT_DIR) 
+      set(CONDUIT_DIR ${AXOM_CONDUIT_DIR}) 
+    endif()
+    find_dependency(Conduit REQUIRED NO_DEFAULT_PATH PATHS "${CONDUIT_DIR}/lib/cmake")
+  endif()
+
+  # hdf5
+  if(AXOM_USE_HDF5)
+    set(AXOM_HDF5_DIR     "@HDF5_DIR@")
+    # Note: Targets not currently imported
+  endif()
+
+  # mfem
+  if(AXOM_USE_MFEM)
+    set(AXOM_MFEM_DIR     "@MFEM_DIR@")
+    # Note: Targets not currently imported
+  endif()
+
+  # raja
+  if(AXOM_USE_RAJA)
+    set(AXOM_RAJA_DIR     "@RAJA_DIR@")
+    if(NOT RAJA_DIR) 
+      set(RAJA_DIR ${AXOM_RAJA_DIR}) 
+    endif()
+    find_dependency(RAJA REQUIRED NO_DEFAULT_PATH PATHS "${RAJA_DIR}/share/raja/cmake")
+  endif()
+
+  # scr
+  if(AXOM_USE_SCR)
+    set(AXOM_SCR_DIR      "@SCR_DIR@")
+    # Note: Targets not currently imported
+  endif()
+
+  # umpire
+  if(AXOM_USE_UMPIRE)
+    set(AXOM_UMPIRE_DIR   "@UMPIRE_DIR@")
+    if(NOT UMPIRE_DIR) 
+      set(UMPIRE_DIR ${AXOM_UMPIRE_DIR}) 
+    endif()
+    find_dependency(umpire REQUIRED NO_DEFAULT_PATH PATHS "${UMPIRE_DIR}/share/umpire/cmake")
+  endif()
+
+  #----------------------------------------------------------------------------
+  # Include targets exported by cmake
+  #----------------------------------------------------------------------------
+  get_filename_component(AXOM_CMAKE_CONFIG_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+  include(${AXOM_CMAKE_CONFIG_DIR}/axom-targets.cmake)
+
+  # HACK: Clear INTERFACE_COMPILE_OPTIONS to avoid requiring OpenMP in user code
```

Comment:
@white238 -- This was a hack to remove openmp flags from our exported target.
This should hopefully be unnecessary after our handling of openmp flags is fixed/improved.

Bonus: Users no longer need to set `ENABLE_FORTRAN=ON` in their build systems just to get `C++` code to compile :)